### PR TITLE
[Fix-9251] [WORKER] reslove the sql task about of add the udf resource failed

### LINKED ISSUE #9251: [Bug] [server] HIVE wokflow,no resource and no log
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

1 ihad bound a udf function to my workflow,but when I run it, it said that the resource is not exists ,and i can't see the log
image

### What you expected to happen

the hive sql can execute and  see the log normal

### How to reproduce

check the code and fix it

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by nobolity**
## Purpose of the pull request

fix #9251 

## Brief change log
change the JsonTypeInfo.type  in AbstractResourceParameters  into resourceType

and add resourceType property to udf ,datasource to serialize the entity 

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Comment by davidzollo** [comment]
please update the PR title to the consistent style, you can refer this one, `[Fix-9221] [alert-server] optimization and gracefully close`

File: dolphinscheduler-common/src/main/resources/common.properties [hunk 1]
```diff
@@ -47,7 +47,7 @@ fs.defaultFS=hdfs://mycluster:8020
 aws.access.key.id=minioadmin
 aws.secret.access.key=minioadmin
 aws.region=us-east-1
-aws.endpoint=http://localhost:9000
+aws.endpoint=http://localhost:9002
```
Comment (lenboo) [inline dolphinscheduler-common/src/main/resources/common.properties]: why this port is changed?

**Review by lenboo — APPROVED** [review]
+1

File: dolphinscheduler-common/src/main/resources/common.properties (hunk 1 above)
Comment (nobolity) [reply dolphinscheduler-common/src/main/resources/common.properties]: yes,had changed

**Pull request merged**